src/main.ts

```diff
@@ -1728,7 +1728,11 @@ I strongly recommend to use "Source mode" for viewing the conflicted files. For
         this.statusBar?.displayMessage(message.toLowerCase(), timeout);
 
         if (!this.settings.disablePopups) {
-            new Notice(message, 5 * 1000);
+            if (!this.settings.disablePopupsForNoChanges
+                || (this.settings.disablePopupsForNoChanges
+                && !message.startsWith("No changes"))) {
```

Comment:
```suggestion
                || !message.startsWith("No changes")) {
```
`disablePopupsForNoChanges` should always be true if the first case isn't met.